tests/Feature/AdminTest.php

```diff
@@ -73,7 +74,7 @@ private function assertCanBanUsers()
         $this->put('/admin/users/'.$user->username().'/ban')
             ->assertRedirectedTo('/admin/users/'.$user->username());
 
-        $this->seeInDatabase('users', ['id' => $user->id(), 'is_banned' => true]);
+        $this->seeInDatabase('users', ['id' => $user->id(), 'banned_at' => Carbon::now()]);
```

Comment:
Your test is failing atm because the `Carbon::now()` slightly differs from the one set when banning the user. It's best solved by checking the opposite of the timestamp being checked: check wether it's not null.

```php
$this->notSeeInDatabase('users', ['id' => $user->id(), 'banned_at' => null]);
```